extreme_classification/hierarchicalXC/classifier.py

```diff
@@ -86,13 +80,14 @@ def predict(self, X):
         Returns:
             classes : the class predictions for each data point
         """
-        X = X.toarray()
+        self.X = X.toarray()
```

Comment:
Sorry about being too anal about this, can you call this self.test_X or sth. This will not cause confusion.

Reply:
Done